java/com/google/flatbuffers/Table.java

```diff
@@ -56,6 +56,23 @@ protected Charset initialValue() {
    */
   public ByteBuffer getByteBuffer() { return bb; }
 
+  public <T> T getScalarField(int field, T defaultValue, Class<T> type) {
+    int offset = __offset(field);
+
+    return type.cast(null);
+  }
+
+  /**
+   * Initialized this {@link Table} to the given position for the giver {@code ByteBuffer}.
+   *
+   * @param position position of this {@link Table} in the {@code ByteBuffer}.
+   * @param bb the {@code ByteBuffer} whose contains this {@link Table}.
+   * @return this {@link Table} initialized.
+   */
+  protected Table _init(int position, ByteBuffer bb) {
```

Comment:
Hi Wouter,
I would like to make this method part of the parent class (i.e. Table) instead of generate it for all the subclasses. Does it make sens for you?
I'm using it in the Reflection.java at: https://github.com/google/flatbuffers/pull/4019/files#diff-fe28cbc55417d497c520cbc9797dbc6eR27